tests/integration/ceph_base_block_test.go

```diff
@@ -395,10 +395,10 @@ func restartOSDPods(k8sh *utils.K8sHelper, s suite.Suite, namespace string) {
 
 	// Delete the osd pod(s)
 	logger.Infof("Deleting osd pod(s)")
-	pods, err := k8sh.Clientset.CoreV1().Pods(namespace).List(metav1.ListOptions{LabelSelector: osdLabel})
+	pods, _ := k8sh.Clientset.CoreV1().Pods(namespace).List(metav1.ListOptions{LabelSelector: osdLabel})
```

Comment:
ditto do not silence the error, handle it

Reply:
Done.